Tools/autotest/pysim/vehicleinfo.py

```diff
@@ -120,6 +120,11 @@ def __init__(self):
                 "waf_target": "bin/arduplane",
                 "default_params_filename": "default_params/quadplane-tilttrivec.parm",
             },
+            "quadplane-tilthvec": {
+                "make_target": "sitl",
+                "waf_target": "bin/arduplane",
+                "default_params_filename": "default_params/quadplane-tilthvec.parm",
```

Comment:
Please consider inheriting the bulk of the parameters from `plane.parm`; you can see how Copter does a lot of that.